README.rst

```diff
@@ -112,7 +112,13 @@ things easier. To run them, first install development requirements::
 
 Run tests using py.test::
 
-    py.test
+    python -m pytest
```

Comment:
Could I clarify the purpose of this change? It doubtless makes sense but isn't clear to me.

Reply:
@bwhicks pytest can't load the plugin that disconnects the index signal handlers unless it is on the python path.  You can set the environment variable (which is what I'm doing), but for convenience, if you run it this way you get the current directory added to your python path.  I'll add a bit more content to clarify that.

This won't be problem once we refactor/generalize this code into a separate module.